llama_index/node_parser/relational/base_element.py

```diff
@@ -14,7 +14,9 @@
 from llama_index.utils import get_tqdm_iterable
 
 DEFAULT_SUMMARY_QUERY_STR = """\
-What is this table about? Give a very concise summary (imagine you are adding a caption), \
+What is this table about? Give a very concise summary (imagine you are adding a new caption and summary for this table), \
+and output the real/existing table title/caption if context provided.\
+and output the real/existing table id if context provided.\
```

Comment:
Is the table ID important for retrieval?

Reply:
will be useful:
1. locate table
2. provide citation/reference